### PR TITLE
Rename recovered bikes to abandoned bikes

### PR DESCRIPTION
**Pull request by Robertgervais**

### CODE REVIEW

**Comment by Robertgervais** [comment]
#1096

File: app/models/bike.rb [hunk 1]
```diff
@@ -80,7 +80,7 @@ class Bike < ActiveRecord::Base
   scope :impounded, -> { includes(:impound_records).where(impound_records: { retrieved_at: nil }).where.not(impound_records: { id: nil }) }
   # "Recovered" bikes are bikes that were found and are waiting to be claimed. This is confusing and should be fixed
   # so that it no longer is the same word as stolen recoveries
-  scope :non_recovered, -> { where(recovered: false) }
+  scope :non_recovered, -> { where(abandoned: false) }
```
Comment (sethherr) [inline app/models/bike.rb]: I'd like this scope to be renamed as well.

File: app/views/bikes/_bike.html.haml [hunk 2]
```diff
@@ -16,7 +16,7 @@
         %ul.attr-list
           %li
             %span.attr-title.text-danger
-              = bike.recovered ? t(".recovered") : t(".stolen")
+              = bike.abandoned ? t(".recovered") : t(".stolen")
```
Comment (sethherr) [inline app/views/bikes/_bike.html.haml]: Could you change the translation key so it matches expectations?

File: app/views/bikes/show.html.haml [hunk 3]
```diff
@@ -141,7 +141,7 @@
                 = @bike.attr_list_item(@stolen_record.address(skip_default_country: true), t(".location"))
                 = @bike.attr_list_item(@stolen_record.locking_description, t(".locking_description"))
                 = @bike.attr_list_item(@stolen_record.lock_defeat_description, t(".locking_circumvented"))
-                - stolen_type = @bike.recovered ? 'recovered' : 'stolen'
+                - stolen_type = @bike.abandoned ? 'recovered' : 'stolen'
```
Comment (sethherr) [inline app/views/bikes/show.html.haml]: Update text here. Probably need to change some of the text as well, in the translation file

File: db/migrate/20190807175348_rename_recovered_to_abandonded.rb [hunk 4]
```diff
@@ -0,0 +1,10 @@
+class RenameRecoveredToAbandonded < ActiveRecord::Migration
+
+  def up
+    rename_column :bikes, :recovered, :abandoned
```
Comment (sethherr) [inline db/migrate/20190807175348_rename_recovered_to_abandonded.rb]: You can do a `change` migration, no need for up and down

File: app/models/bike.rb (hunk 1 above)
Comment (sethherr) [reply app/models/bike.rb]: 🗣

**Pull request merged**